net-setup.sh

```diff
@@ -16,7 +16,8 @@
 
 usage() {
     cat <<EOF
-$0 [--config|-c <file>] [--iface|-i <interface>] [start|up] [stop|down]
+$0 [--config|-c <file>] [--iface|-i <interface>] [--bridge|-b <bridge>]
```

Comment:
Feedback from slack:

- Keep net-setup.sh more generic, move (bridge-)specific stuff to conf file.
- maybe remove bridge afterwards
- forward all parameters $@ to conf file